board/main.c

```diff
@@ -153,11 +153,13 @@ int usb_cb_ep1_in(uint8_t *usbdata, int len, bool hardwired) {
 // send on serial, first byte to select the ring
 void usb_cb_ep2_out(uint8_t *usbdata, int len, bool hardwired) {
   UNUSED(hardwired);
-  if (len == 0) return;
-  uart_ring *ur = get_ring_by_number(usbdata[0]);
-  if (!ur) return;
-  if ((usbdata[0] < 2) || safety_tx_lin_hook(usbdata[0]-2, usbdata+1, len-1)) {
-    for (int i = 1; i < len; i++) while (!putc(ur, usbdata[i]));
+  if (len != 0) {
+    uart_ring *ur = get_ring_by_number(usbdata[0]);
+    if (ur != NULL) {
```

Comment:
merge 156 and 158 to avoid deep indent?

Reply:
yep, I'll move `uart_ring *ur = get_ring_by_number(usbdata[0]);` up